reflect.go

```diff
@@ -12,15 +12,19 @@ import (
 var reflectTypeType = reflect.TypeOf((*reflect.Type)(nil)).Elem()
 
 // ReflectTypeFilter is a filter that formats reflect.Type values.
-func ReflectTypeFilter(w io.Writer, v Value) (n int, err error) {
+func ReflectTypeFilter(
+	w io.Writer,
+	v Value,
+	format func(w io.Writer, v Value),
+) (written bool, err error) {
```

Comment:
```suggestion
) (_ bool, err error) {
```

I know `err` has to be captured to use `must.Recover()`, but I'd prefer regular return statements rather than bare-returns throughout the function body.